### DISCUSSION TITLE
Deformable coregistration mode

### DISCUSSION THREAD

**Discussion by smcch** [CBICA/CaPTk#1507] [Q&A] [answered]
What is the algorithm used by CaPTK for the deformable registration mode?

Thanks

**Comment by sarthakpati** [comment, marked as answer]
Hi,

We use the [Greedy algorithm](https://greedy.readthedocs.io/en/latest/index.html) for registration. You can find more information on this here: https://cbica.github.io/CaPTk/preprocessing_reg.html

Cheers,
Sarthak

**Comment by smcch** [comment]
Thanks!
